roles/common/files/uug_ansible_wrapper.py

```diff
@@ -388,7 +388,10 @@ def on_run_clicked(self, _):
         if not validate_branch():
             invalid_branch(self)
             return
-
+        
+        if not validate_config_settings(self):
```

Comment:
Since this is supposed to just be a warning, let's still continue even if it fails.